src/DynamoUtilities/PathHelper.cs

```diff
@@ -133,18 +133,18 @@ public static bool isValidJson(string path, out string fileContents)
                     var obj = Newtonsoft.Json.Linq.JToken.Parse(fileContents);
                     return true;
                 }
-                return false;
+                throw new JsonReaderException();
```

Comment:
Same comment here as above, throwing instead of returning a bool. Just want to make sure the change in implementation is expected.